.circleci/config.yml

```diff
@@ -308,13 +308,12 @@ jobs:
           command: scripts/deploy.sh "$ENV_TARGET" wedlocks netlify "$CIRCLE_SHA1" "$CIRCLE_BRANCH" "$IS_FORKED_PR"
 
   deploy-locksmith-beanstalk:
-    machine: true
+    docker:
+      - image: circleci/python:3.6.4
     environment:
       DOCKER_REPOSITORY: unlockprotocol
     steps:
       - checkout
-      - run:
-          command: scripts/monorepo.sh locksmith $CIRCLE_BRANCH
```

Comment:
Do you mean to run this on every merge?

Reply:
thanks, this is listed as a draft pr for this reason. When toggled as a standard PR with will be removed. This configuration change can not be tested in a branch without removing these lines